packages/app/helper/useWorkoutVideo.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react'
+import { getDoc, doc } from 'firebase/firestore'
+import { db } from 'app/firebase/client'
+
+export function useWorkoutVideo(videoId) {
+  const [video, setVideo] = useState([])
+  const videoDoc = doc(db, 'video', videoId)
+
+  useEffect(() => {
+    const fetchSession = async () => {
+      const videoSnapshot = await getDoc(videoDoc)
+      if (videoSnapshot.exists()) {
+        const videoData = videoSnapshot.data()
+        console.log('videoData :', videoData)
+        const vimeoRes = await fetch(
+          //`http://localhost:3000/api/vimeo/${videoData.videoId}`
```

Comment:
please remove and maube put the BE url into an ENV variable

Reply:
We didn't put it in an env var because react native can't use env vars. They have their own system. Our (unsafe) solution was to import all env vars from a js file. Due to a cors error we always need to switch between the two tho...